openshift/launchpad-template.yaml

```diff
@@ -41,11 +41,13 @@ parameters:
 - description: OpenShift user name (leave empty when using Keycloak)
   displayName: OpenShift user name
   name: LAUNCHPAD_MISSIONCONTROL_OPENSHIFT_USERNAME
-  value: developer
+  value: 
+  required: true
```

Comment:
I just read your explanation for this on the mailing list @jorgemoralespou . I understand using default username/password in a template is not best practice, but I'm not sure if ease-of-use is more important here or not ( @ALRubinger ? ). They still need to be optional though.